sql/src/test/java/io/crate/integrationtests/SysShardsTest.java

```diff
@@ -316,5 +316,9 @@ public void testSelectNodeSysExpressionWithUnassignedShards() throws Exception {
         String nodeName = response.rows()[0][0].toString();
         assertEquals("node_0", nodeName);
         assertThat(response.rows()[8][0], is(nullValue()));
+        transportExecutor.exec(
+                "drop table users"
+        );
+        transportExecutor.ensureGreen();
```

Comment:
wrap the whole stuff into try finally block. otherwise the other tests will still fail if this test fails